jsapp/js/components/processing/sidebar/sidebarSubmissionMedia.tsx

```diff
@@ -36,19 +36,27 @@ export default function SidebarSubmissionMedia(props: SidebarSubmissionMediaProp
           `}
           key='audio'
         >
-          <AudioPlayer mediaURL={attachment.download_url} filename={attachment.filename} />
-        </section>
-      )
-    case QUESTION_TYPES.video.id:
-      return (
-        <section
-          className={`
-            ${styles.mediaWrapper}
-            ${styles.mediaWrapperVideo}
-          `}
-          key='video'
-        >
-          <video className={styles.videoPreview} src={attachment.download_url} controls />
+          <AudioPlayer
+            mediaURL={attachment.download_url}
+            filename={attachment.filename}
+            rightHeaderSection={
+              store.data.submissionData && (
+                <AttachmentActionsDropdown
+                  asset={props.asset}
+                  submissionData={store.data.submissionData}
+                  attachmentId={attachment.id}
+                  onDeleted={() => {
+                    // TODO: this might be done with a bit more elegant UX, as calling the function causes a whole page
```

Comment:
:pray: comment answered most of the questions I had. I agree with the best approach being to make the store use react-query. Maybe we should create a task for this in post mvp?

Reply:
BTW, would we encounter strange behaviour if we remove the audio attachment from the NLP view? That's the only way we can get to through the UI IIRC. It would definitely show a blank box after rerender so we should set it to disable the audio in the sidebar (and disable the switch in settings).

We should also explain the consequences of deleting the audio from the NLP screen in the delete modal

Reply:
I think that deleting audio attachment in the context of NLP should only influence the sidebar player. So instead of player, we should display `DeletedAttachment` in there. So there should be some logic inside `SidebarSubmissionMedia` for it. We have to remember that user can have transcript and translations etc. and deleting audio shouldn't block the access to all that :) I imagine some users would delete attachment immediately after getting transcript

Reply:
As for creating a task, I will ask Jackie where she would like us to put them :)